pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator.go

```diff
@@ -34,7 +36,15 @@ func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admiss
 		return util.ToAdmissionResponseError(err)
 	}
 
-	if providerType, ok := secret.GetLabels()["createdForProviderType"]; ok && providerType == "ovirt" {
+	if _, ok := secret.Data["insecureSkipVerify"]; ok {
+		insecure, err = strconv.ParseBool(string(secret.Data["insecureSkipVerify"]))
```

Comment:
so why not to take the the value of secret.Data["insecureSkipVerify"] from the line above?